README.md

```diff
@@ -1,6 +1,13 @@
 # Dev Joke
 #### Send me a PR if you know a good Dev Joke.
 
+![lolol](./images/lolol.jpg)
+* * *
+Q: what from is bat?
```

Comment:
can be reworded to "Where does bat come from?"

Reply:
okay, can be. it was just that here was nothing about bats and bcos of that. not that there mistake or smth. just about bat, so you can change it, yep. =)